packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
+## 5.2.3
```

Comment:
Blank line between heading and list to match all the other entries please.

Reply:
done